swagger.yml

```diff
@@ -6,6 +6,62 @@ info:
 
 
 paths:
+  /operations:
+    get:
+      summary: List available operations
+      description: >
+        Retrieve a list of all available operations on this server.
+        The return value is an object whose attributes are operation names,
+        and each attribute value is the description of the arguments
+        the argument takes.
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                example: >
+                    {
+                       "FromBase64" : {
```

Comment:
Please can you define the example response in YAML rather than a string that looks like JSON please. there should be an example of this in `swagger.yml` now that the magic operation docs are included